### DISCUSSION TITLE
Appimage self-update functionality

### DISCUSSION THREAD

**Discussion by gc-user** [rustdesk/rustdesk#10083] [General]
A few of the appimages I use apparently have inbuilt the functionality to check, download and "install" (they just replace the old appimage file with the new and rename the downloaded appimage to whatever a user named the existing appimage file) updates at start if an update is available and the use clicks the "update now" button offered in the notification about the available update.

That would be a very helpful feature to have because then having to use another remote desktop software to update the rustdesk client on the remote side would be thing of the past.

Of course, this would be a feature that all client variants would benefit from, but apart from appimages and applications like firefox are the only ones I know do this, so I can't speak to other linux variants of rustdesk, or even windows or mac versions.

This might be what I mean:

[Making AppImages self-updateable](https://docs.appimage.org/packaging-guide/optional/updates.html#making-appimages-self-updateable)

**Comment by xlionjuan** [comment]
Use this

https://flathub.org/apps/it.mijorus.gearlever
